src/cascadia/TerminalControl/TermControl.h

```diff
@@ -23,19 +23,28 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         public CopyToClipboardEventArgsT<CopyToClipboardEventArgs>
     {
     public:
-        CopyToClipboardEventArgs(hstring text, hstring html, hstring rtf) :
+        CopyToClipboardEventArgs(hstring text) :
+            _text(text),
+            _html(),
+            _rtf(),
+            _formats(Settings::CopyFormat::Plain) {}
+
+        CopyToClipboardEventArgs(hstring text, hstring html, hstring rtf, Windows::Foundation::IReference<Settings::CopyFormat> formats = nullptr) :

```

Comment:
are you ever calling this without a fourth parameter?

Reply:
remind my why html/rtf being _empty_ isn't good enough to indicate that the format was not selected?

Reply:
Whoops! That must've been left over from one of my previous iterations. My bad.

> remind my why html/rtf being empty isn't good enough to indicate that the format was not selected?

actually, we can totally do that in CopyToClipboardEventArgs. IIRC the problem was more just passing CopyFormat down (TermPage --> TermControl). Updating now.